### PR TITLE
Update privacy policy

### PR DESCRIPTION
**Pull request by poorvasingal**
Includes changes for regional partners + teacher forum. Related to https://github.com/code-dot-org/code-dot-org/pull/18425.

### CODE REVIEW

**Comment by poorvasingal** [comment]
@tanyaparker - Preview is showing me the following but I'm not sure what I'm doing wrong...
image

I didn't change the first line, but it's showing the "\." And there is a line break between the two paragraphs but that doesn't seem to be showing up either. Ideas?

File: pegasus/sites.v3/code.org/public/privacy.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Our Privacy Policy is fueled by our commitment to the following Privacy Principl
...
+We are committed to creating a safe and secure environment for learners of all ages on our websites located at [https://code.org](https://code.org), [https://studio.code.org](https://studio.code.org), [https://csedweek.org](https://csedweek.org), [https://hourofcode.com](https://hourofcode.com), [https://forum.code.org](https://forum.code.org), other websites, application programming interface (our “API”), and online services (collectively, our “Website”). Visitors and users of the Website and participants in our education programs are referred to individually as “User” and collectively as “Users”. As a not-for-profit, 501(c)(3) organization, we use the data we receive only insofar as it helps our mission of providing a great computer science education for every student who wants it. We established ourselves as a not-for-profit organization so that a for-profit motive will not interfere with our mission of providing a trusted educational resource.
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/privacy.md]: Should we be including curriculum.code.org too?

File: pegasus/sites.v3/code.org/public/privacy.md [hunk 2]
```diff
@@ -63,16 +63,18 @@ Written comments in response to curricular/educational prompts within Code.org c
...
-Survey and demographic data | Optional | For the purposes of evaluating our own work and improving our education results, Code.org regularly sends surveys to teachers. <br><br>These surveys are completely optional. The data provided by teachers in these surveys is saved and used for analysis by Code.org or by our Evaluator or our professional learning partners or facilitators. Any survey data shared with external parties will be anonymous and aggregated.
+Survey and demographic data | Optional | For the purposes of evaluating our own work and improving our education results, Code.org regularly sends surveys to teachers. <br><br>These surveys are completely optional. The data provided by teachers in these surveys is saved and used for analysis by Code.org or by our Evaluator or our Regional Partners or facilitators. Any survey data shared with external parties will be de-identified and aggregated.
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/privacy.md]: Should this be a list now? And I've heard that we should always put Code.org in front of Facilitators and Regional Partners for official docs?

`for analysis by Code.org, Code.org's Evaluator, Code.org Regional Partners, or Code.org Facilitators.`

File: pegasus/sites.v3/code.org/public/privacy.md [hunk 3]
```diff
@@ -192,11 +194,16 @@ In particular, we do not share any Personal Information you provide with our don
...
 
 In the context of partnerships between Code.org and school districts, we may also share this workshop attendance and professional learning data with the school district that employs the teacher, in order to allow school districts to compensate teachers in situations where school districts have such an arrangement with their teachers.
 
+### We may share de-identified information on school or district usage with our Regional Partners
+
+Our Regional local Partners will be able to see de-identified demographic information and usage of courses and professional learning programs for schools and districts in their area. They will not see names or contact information of any teacher unless you choose to share it as part of signing up for a local professional development workshop or joining their professional learning program.
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/privacy.md]: Is there a reason local was put in the middle? I think we should keep the official name together like `Our local Regional Partners...`

Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/privacy.md]: Consistency between professional development and professional learning?

File: pegasus/sites.v3/code.org/public/privacy.md (hunk 2 above)
Comment (poorvasingal) [reply pegasus/sites.v3/code.org/public/privacy.md]: "Regional Partner" has a definition in our privacy policy now so I think we should leave it like that. We also say simply "facilitators" in the policy, so I think just keeping it like that is probably also fine. I am updating to make it more like a list like you suggested now though.

File: pegasus/sites.v3/code.org/public/privacy.md (hunk 3 above)
Comment (poorvasingal) [reply pegasus/sites.v3/code.org/public/privacy.md]: Whoops, I meant to take the local out. Thanks! And Alice wrote it up like this and I think I remember her telling me it was intentional...so I might leave it as "professional development" for now.

**Pull request merged**